src/templates/homepage.html

```diff
@@ -1,10 +1,10 @@
 <div class="app-preview"></div>
 
 <div class="app-list">
-  {% defer (url=api('apps'), pluck='objects', as='apps', key='slug') %}
+  {% defer (url=api('apps'), pluck='objects', as='apps', key='id') %}
     <ul class="clearfix">
       {% for app in this %}
-        <li class="app-list-app focusable" data-slug="{{ app.slug }}">
+        <li class="app-list-app focusable" data-id="{{ app.id }}">
```

Comment:
The app install logic might look for data-slug. Have you tried whether apps still install?